### ISSUE TITLE
What kind of accuracy logic is used in this Reference Kit?

### ISSUE THREAD

**Issue by tarekradi** [oneapi-src/visual-quality-inspection#8]
This Reference Kit does the following import..

from neural_compressor.experimental import Benchmark

then it runs the following code...

    evaluator = Benchmark(config_path)
    evaluator.model = int8_model
    # create benchmark dataloader like examples/tensorflow/qat/benchmark.py
    evaluator.b_dataloader = test_loader
    evaluator('accuracy')

    print("*"*50)
    print("Evaluating the FP32 Model")
    print("*"*50)
    evaluator = Benchmark(config_path)
    evaluator.model = model
    # create benchmark dataloader like examples/tensorflow/qat/benchmark.py
    evaluator.b_dataloader = test_loader
    evaluator('accuracy')

What kind of accuracy logic is being run behind the scenes?

**Comment by huiyan2021** [comment]
The accuracy metric is defined in yaml file: https://github.com/oneapi-src/visual-quality-inspection/blob/main/src/intel_neural_compressor/conf.yaml#L8

**Comment by tarekradi** [comment]
Sorry, but i still don't know the answer. 
That line says it is optional but required if the user doesn't provide eval_func in neural_compressor.Quantization.

I didn't change anything in the code. What is the accuracy metric used by default in this reference kit?

**Comment by huiyan2021** [comment]
https://github.com/oneapi-src/visual-quality-inspection/blob/main/src/intel_neural_compressor/conf.yaml#L10
  accuracy:         
    metric:
      topk: 1
Is this the answer that you need?
